### PR TITLE
[Docs][UI Next][V1.0.0-Alpha] Add license file.

### PR DESCRIPTION
**Pull request by songjianet**
## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

### CODE REVIEW

**Review by CalvinKirs — COMMENTED** [review]
`https://github.com/apache/dolphinscheduler/tree/dev/dolphinscheduler-dist/release-docs/licenses` You seem to be missing something here?

**Comment by songjianet** [comment]
The file corresponding to the `license` is already in `dolphinscheduler-dis/release-docs/licenses/ui-next-licenses` .

File: dolphinscheduler-dist/release-docs/LICENSE [hunk 1]
```diff
@@ -584,3 +584,64 @@ Apache 2.0 licenses
...
+
+
+========================================================================
+UI-Next related licenses
+========================================================================
+
+The following components are used in UI-Next.See project link for details.
+The text of each license is also included at licenses/ui-next-licenses/LICENSE-[project].txt.
+
+========================================
```
Comment (kezhenxu94) [inline dolphinscheduler-dist/release-docs/LICENSE:596]: Seems you miss a section for those fonts `SIL OPEN FONT LICENSE`?

Comment (songjianet) [reply dolphinscheduler-dist/release-docs/LICENSE:596]: I did ignore the font problem, but the font does not have a corresponding version number, nor does it seem to have a corresponding github repository address or npm address. How to deal with this.

Comment (kezhenxu94) [reply dolphinscheduler-dist/release-docs/LICENSE:596]: FiraCode has a repo and version: https://github.com/tonsky/FiraCode
https://github.com/mozilla/Fira

Version numbers are not a must in this file, but this file serves as a "Table of Content" for the licenses, "The text of each license is also included at licenses/ui-next-licenses/LICENSE-[project].txt.", if the licenses are not listed in this file, users might ignore those licenses and don't check files under `licenses/.........`

Comment (songjianet) [reply dolphinscheduler-dist/release-docs/LICENSE:596]: OK, I'll deal with it now.

**Review by kezhenxu94 — APPROVED** [review]

**Pull request merged**